docs/src/main/paradox/kubernetes-deployment/deploying.md

```diff
@@ -0,0 +1,124 @@
+# Deploying
+
+Having configured a DeploymentSpec, Role, and RoleBinding they can be created with:
```

Comment:
isn't the namespace needed?
```
kubectl apply -f kubernetes/namespace.json
kubectl config set-context --current --namespace=appka-1
```

Reply:
The namespace is in the DeploymentSpec so no need to specify at the command line: https://github.com/akka/akka-sample-cluster-kubernetes-java/blob/master/kubernetes/akka-cluster.yml#L7